burn-train/src/learner/early_stopping.rs

```diff
@@ -0,0 +1,185 @@
+use crate::metric::{
+    store::{Aggregate, Direction, EventStoreClient, Split},
+    Metric,
+};
+
+/// The condition that [early stopping strategies](EarlyStoppingStrategy) should follow.
+pub enum StoppingCondition {
+    /// When no improvement has happened since the given number of epochs.
+    /// In other words, when no best epoch has been found.
```

Comment:
I feel like this second sentence is just confusing. The condition is already clear from the first sentence, and there's always a "best" epoch even if it stagnates